tests/data/faulty_html/Bill_Gates.html

```diff
@@ -0,0 +1,310 @@
+<!DOCTYPE html>
```

Comment:
Is it possible to makes these example docs minimal (just to reduce the unnecessary lines of code).